src/Microsoft.AspNet.Mvc.ViewFeatures/Rendering/Html/DefaultHtmlGenerator.cs

```diff
@@ -153,7 +154,7 @@ public virtual HtmlString GenerateAntiforgery([NotNull] ViewContext viewContext)
             ModelExplorer modelExplorer,
             string expression)
         {
-            var tagBuilder = new TagBuilder("input", _htmlEncoder);
```

Comment:
Are the `Encode()` methods still used anywhere? If not, can get rid of the `_htmlEncoder` field and related constructor parameter entirely. Fortunately there are very few places where we invoke that constructor.